src/gui/optionsdialog.ui

```diff
@@ -17,32 +17,32 @@
    <item>
     <widget class="QSplitter" name="hsplitter">
      <property name="orientation">
-      <enum>Qt::Horizontal</enum>
+      <enum>Qt::Orientation::Horizontal</enum>
```

Comment:
It seems this is a global change. it should be changed for all .ui files.
Do you mind separating it out to its own commit?

Reply:
>Do you mind separating it out to its own commit?

Very much actually, that's way too much work.

Reply:
@Chocobo1 Do you mean all instances of `Qt::Horizontal` should be changed to `Qt::Orientation::Horizontal` in *.ui files & perhaps same for `Qt::Vertical` -> `Qt::Orientation::Vertical`??

Reply:
>It seems this is a global change. it should be changed for all .ui files.

I'm doing this now.

>Do you mean all instances of Qt::Horizontal should be changed to Qt::Orientation::Horizontal in *.ui files & perhaps same for Qt::Vertical -> Qt::Orientation::Vertical??

Designer does all such changes automatically.